services/src/main/java/io/druid/cli/validate/DruidJsonValidator.java

```diff
@@ -52,6 +64,7 @@ public void run()
     }
 
     final ObjectMapper jsonMapper = new DefaultObjectMapper();
+    final ClassLoader loader = DruidJsonValidator.class.getClassLoader();
```

Comment:
Suggest checking thread's context class loader and using  that first, and falling back to this if thread's context class loader is null.